Top/main.c

```diff
@@ -575,7 +575,7 @@ static int32_t csoundCompileCSDText(CSOUND *csound, const char *csd_text, int32_
     if (csound->csdname != NULL)
       csound->Free(csound, csound->csdname);
     csound->csdname = cs_strdup(csound, "*string*"); /* Mark as from text. */
-    res = csoundCompileOrcInternal(csound, NULL, async);
+    res = csound_compile_orc(csound, NULL, async);
```

Comment:
@vlazzarini I merged latest from develop and modified this line to pass async to csound_compile_orc. Prior to this in the PR branch, the 3rd arg was set to 0. Can you confirm this is okay to pass async here?

Reply:
yes that is the idea. Before, it was hardwired to 0, now it's possible to change it.